pkg/cli/alpha/config-gen/cert-generation-filter.go

```diff
@@ -107,6 +107,13 @@ func (c CertFilter) Filter(input []*yaml.RNode) ([]*yaml.RNode, error) {
 		if err != nil {
 			return nil, err
 		}
+		// TODO: verify if this works, since `yaml.Append` requires a node to be created for clientConfig
+		err = matches[i].PipeE(yaml.LookupCreate(
+			yaml.SequenceNode, "spec", "conversion", "webhookClientConfig", "conversionReviewVersions"),
+			yaml.FieldSetter{StringValue: "v1"})
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Thought about this a bit more. This filter seems to only handle v1beta1 CRDs, which have a default conversion review version list containing "v1beta1". Therefore you can just remove this change.